renku/core/commands/workflow.py

```diff
@@ -45,100 +49,69 @@ def _deref(ref):
     return ref[len("workflows/") :]
 
 
-@inject.autoparams()
-def _list_workflows(client: LocalClient):
-    """List or manage workflows with subcommands."""
-    from renku.core.models.refs import LinkReference
-
-    names = defaultdict(list)
-    for ref in LinkReference.iter_items(common_path="workflows"):
-        names[ref.reference.name].append(ref.name)
-
-    for path in client.workflow_path.glob("*.yaml"):
-        communication.echo(
-            "{path}: {names}".format(path=path.name, names=", ".join(_deref(name) for name in names[path.name]))
-        )
-
-
-def list_workflows_command():
-    """Command to list or manage workflows with subcommands."""
-    return Command().command(_list_workflows).require_migration().with_database()
+def _to_view(workflow: AbstractPlan) -> Union[CompositePlanViewModel, PlanViewModel]:
+    if isinstance(workflow, CompositePlan):
+        return CompositePlanViewModel.from_composite_plan(workflow)
+    return PlanViewModel.from_plan(workflow)
 
 
 @inject.autoparams()
-def _set_workflow_name(name, path, force, client: LocalClient):
-    """Sets the <name> for remote <path>."""
-    from renku.core.models.refs import LinkReference
-
-    LinkReference.create(client=client, name=_ref(name), force=force).set_reference(path)
+def _find_workflow(name_or_id: str, plan_gateway: IPlanGateway) -> AbstractPlan:
```

Comment:
@Panaetius same as above. one could argue that this is not the best place to define this utility function